example_project/assets/js/app.js

```diff
@@ -27,6 +27,17 @@ import "../css/app.css"
 import components from "../vue"
 import "vite/modulepreload-polyfill"
 
+// Vuetify
+import "vuetify/styles"
```

Comment:
I just checked and it looks like Vuetify causes lots of conflicts with tailwind https://github.com/tailwindlabs/tailwindcss/issues/465

That's the reason why in your version eg. page is white and not dark, and some weird layout shifts after load. 

I'll disable these styles since without them calendar is still working rather fine.